providers/apis/polymarket/README.md

```diff
@@ -0,0 +1,86 @@
+# PolyMarket Provider
+
+Docs: https://docs.polymarket.com/
+
+Polymarket is a web3 based prediction market. This provider uses the Polymarket CLOB REST API to access the off-chain order book. 
+
+## How it Works
+
+Polymarket uses [conditional outcome tokens](https://docs.gnosis.io/conditionaltokens/), a token that represents an outcome of a specific event. All tokens in Polymarket are denominated in terms of USD.
+
+Tickers take the form of:
+
+`<POLYMARKET_TOKEN_ID>/USD`
+
+example: `21742633143463906290569050155826241533067272736897614950488156847949938836455/USD`
```

Comment:
I'm wondering if it's best to make this more human readable. The polymarket token ID is stored in the `off_chain_ticker` so there's nothing preventing us from making this like `polymarket-{market-short-name}/USD` such as `polymarket-us-presidential-election/USD` or something like that.

Reply:
updated [change ticker](https://github.com/skip-mev/slinky/pull/629/commits/eb135bddf1a9feb1288e254e9a2e454116d02333)